.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   extends: ['./src/typescript.js'],
-  ignorePatterns: ['test/fixtures/*'],
```

Comment:
The idea to ignore this folder is to avoid false positives in the linting from this project (`.eslintrc.js` is not something we export), so we can create more complex tests in the future that will check for specific errors. Any reason to remove this line here?

Reply:
It was generating false positives in the tests, but I fixed it by adding `--no-ignore` to the test command.

Another reason is that if the file will be tested, so I thought that will be good to be _linted_ too.
But, I understand that in the future more complex tests will come and this lint enabled will be a problem.

I reverted this change!